bidadjustment/build_rules_test.go

```diff
@@ -427,15 +413,96 @@ func TestMerge(t *testing.T) {
 			givenRequestWrapper: &openrtb_ext.RequestWrapper{
 				BidRequest: &openrtb2.BidRequest{Ext: []byte(`{"ext":{"bidder": {}}}`)},
 			},
-			givenAccount:           &config.Account{},
+			acctBidAdjustments:     nil,
 			expectedBidAdjustments: nil,
 		},
 		{
 			name: "NilAcctBidAdj",
 			givenRequestWrapper: &openrtb_ext.RequestWrapper{
 				BidRequest: &openrtb2.BidRequest{Ext: []byte(`{"prebid":{"bidadjustments":{"mediatype":{"banner":{"bidderA":{"dealId":[{ "adjtype": "multiplier", "value": 1.1}]}}}}}}`)},
 			},
-			givenAccount: &config.Account{},
+			acctBidAdjustments: nil,
+			expectedBidAdjustments: &openrtb_ext.ExtRequestPrebidBidAdjustments{
+				MediaType: openrtb_ext.MediaType{
+					Banner: map[openrtb_ext.BidderName]openrtb_ext.AdjustmentsByDealID{
+						"bidderA": {
+							"dealId": []openrtb_ext.Adjustment{{Type: AdjustmentTypeMultiplier, Value: 1.1}},
+						},
+					},
+				},
+			},
+		},
+
```

Comment:
Nitpick: can we remove the extra space?

Reply:
Removed extra line breaks, the next removal will lead to poor readability

Reply:
I'm sorry for the confusion. By extra space I meant line 435 only. I was ok with the rest of the line breaks. I agree that removing them leads to poor readability. I'll approve anyways.